### PR TITLE
SPUI-256 > Feature/market details services

### PR DESCRIPTION
**Pull request by VovaMedvediev**

### CODE REVIEW

File: lib/service/services/signal_r/model/base_prices_model.dart [hunk 1]
```diff
@@ -0,0 +1,43 @@
+import 'package:freezed_annotation/freezed_annotation.dart';
+
+part 'base_prices_model.freezed.dart';
+part 'base_prices_model.g.dart';
+
+@freezed
+class BasePricesModel with _$BasePricesModel {
+  const factory BasePricesModel({
+    @JsonKey(name: 'priceRecords')
+        required List<InstrumentPriceModel> basePrices,
```
Comment (eli1stark) [inline lib/service/services/signal_r/model/base_prices_model.dart:10]: formatting

File: lib/service/services/chart/model/candles_request_model.dart [hunk 2]
```diff
@@ -6,11 +6,12 @@ part 'candles_request_model.g.dart';
 @freezed
 class CandlesRequestModel with _$CandlesRequestModel {
   const factory CandlesRequestModel({
-    required String instrumentId,
+    @JsonKey(ignore: true) String? instrumentId,
+    @JsonKey(ignore: true) int? type,
```
Comment (eli1stark) [inline lib/service/services/chart/model/candles_request_model.dart:10]: Why? `@JsonKey(ignore: true)`

File: lib/service/services/chart/model/last_candles_request_model.dart [hunk 3]
```diff
@@ -0,0 +1,18 @@
+import 'package:freezed_annotation/freezed_annotation.dart';
+
+part 'last_candles_request_model.freezed.dart';
+part 'last_candles_request_model.g.dart';
+
+@freezed
+class LastCandlesRequestModel with _$LastCandlesRequestModel {
+  const factory LastCandlesRequestModel({
+    @JsonKey(ignore: true) String? instrumentId,
+    @JsonKey(ignore: true) int? type,
```
Comment (eli1stark) [inline lib/service/services/chart/model/last_candles_request_model.dart:10]: Why? `@JsonKey(ignore: true)`

**Review by eli1stark — CHANGES_REQUESTED** [review]

File: lib/service/services/signal_r/model/base_prices_model.dart (hunk 1 above)
Comment (VovaMedvediev) [reply lib/service/services/signal_r/model/base_prices_model.dart:10]: these 2 lines are more then 80 characters so formatting is correct

File: lib/service/services/chart/model/candles_request_model.dart (hunk 2 above)
Comment (VovaMedvediev) [reply lib/service/services/chart/model/candles_request_model.dart:10]: discussed in dm

File: lib/service/services/chart/model/last_candles_request_model.dart (hunk 3 above)
Comment (VovaMedvediev) [reply lib/service/services/chart/model/last_candles_request_model.dart:10]: discussed in dm

**Pull request merged**